.prettierignore

```diff
@@ -0,0 +1 @@
+test/
```

Comment:
prettier is nice, but we may want to keep our test code organized the ways they are (keeping the it.each data vertically aligned, for instance...).